### ISSUE TITLE
mktemp is have a copileft license

### ISSUE THREAD

**Issue by Shatur** [thscharler/spreadsheet-ods#36]
I noticed that this create uses `mktemp` which have MPL license. But this crate is released under MIT/Apache.
But these licenses are not compatible, MIT is more restrictive. I'm not a lawyer, but I'm not sure if it's possible to use it like this.

**Comment by thscharler** [comment]
afaik it should be ok

**Comment by Shatur** [comment]
Closing then, just wanted to make sure that you are aware.

**Comment by Shatur** [comment]
I just noticed that Bevy, for example, deny copyleft licenses because its license is not copyleft:
https://github.com/bevyengine/bevy/blob/ebdf5063df5d2b89f4bf94b52eef03ad027fb8e3/deny.toml#L13

**Comment by Shatur** [comment]
> But this deny.toml looks interesting, is this a standard cargo feature or some extension. Never seen this before?

It's a separate tool called `cargo-deny`:
https://github.com/EmbarkStudios/cargo-deny

**Comment by Shatur** [comment]
No longer used in the crate, closing.

**Comment by thscharler** [comment]
Yes, I did some work recently, and found I no longer need this. 
I think it was some workaround a bug in the zip crate, but I'm no longer shure.
Anyway it's gone now ...
